src/entity/posts/utils.ts

```diff
@@ -285,20 +305,24 @@ export const createExternalLink = async (
         visible: isVisible,
       },
     });
-    await createSharePost(
-      entityManager,
+    const post = await createSharePost({
```

Comment:
Why do we need it returned now?

Reply:
We now need to return it. This makes us reuse the same function. Previous usages should not matter whether this returns anything or not.

Reply:
Because the function is now used on two parts, and the other one needs to utilize some prop from the created post. Specifically the id.